.github/workflows/create-release.yml

```diff
@@ -29,61 +29,88 @@ env:
 
 
 jobs:
-  create-release:
+  generate-requirements-freeze:
     runs-on: ubuntu-latest
     steps:
 
     - name: Checkout spinalcordtoolbox (main branch)
       uses: actions/checkout@v2
-      with:
-        ref: ${{ env.MAIN_BRANCH }}
-        fetch-depth: 0
```

Comment:
This is the change that fixes the issue observed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3802#issuecomment-1139926421.